app/src/main/java/com/sma2/sma2/FeatureExtraction/Movement/WalkingFeatureActivity.java

```diff
@@ -127,4 +146,454 @@ private void onButtonBack(){
         startActivity(i);
     }
 
+    //--------START FUNCTIONS FOR TREMOR---------------------------------------------------
+    // Compute Postural Tremor for both hands
+    // Left Hand
+    private float CalculatePosturalTremorLeft(){
```

Comment:
Move this function to MovementProcessing

This activity only should be used for calling the function and visualization purposes

Reply:
The input to the function should be the path to the file, and it should return the value of the tremor